en/micro-integrator/docs/references/connectors/fhir-connector/fhir-connector-example.md

```diff
@@ -0,0 +1,318 @@
+# FHIR Connector Example
+
+Given below is a sample scenario that demonstrates how to work with resources in [FHIR](https://www.hl7.org/fhir/overview.html)(Fast Healthcare Interoperability Resources). FHIR specification is a standard for exchanging health care information electronically.The WSO2 EI FHIR connector uses the FHIR RESTFul API to interact with FHIR.
+
+## What you'll build
+
+Given below is a sample API that illustrates how you can connect to a FHIR with the `init` operation and then use the `create`,`readResource`, `readSpecificResourceById`,`delete` and `update` operation to send messages to the FHIR server.
+
+* `<fhir.init>` : It will initialize the FHIR operations.
+* `<fhir.create>` : To creates a new resource in a server-assigned location, use `fhir.create` operation.
+* `<fhir.readResource>` : To accesses the current contents of a resource, use `fhir.readResource` operation.
+* `<fhir.readSpecificResourceById>` : To accesses the specific current contents of a resource, use `fhir.readSpecificResourceById` operation.
+* `<fhir.update>` : To creates a new current version for an existing resource or creates an initial version if no resource already exists for the given id, use `fhir.update` operation.
+* `<fhir.update>` : To removes an existing resource, use `fhir.delete`operation.
+
+To know the further information about these operations please refer this link.
+
+> **Note**:If no id element is provided, or the value is wrong, the server SHALL respond with a HTTP 400 error code, and SHOULD provide an operation outcome identifying the issue.
+
+The following diagram illustrates all the required functionality of the FHIR API service that you are going to build.
+
+<img src="/assets/img/connectors/FHIRConnector.png" title="FHIR Connector" width="800" alt="FHIR Connector"/>
```

Comment:
Add some content to describe this diagram and what is happening in this scenario.

Reply:
Added.